core/domain/state_domain.py

```diff
@@ -2151,3 +2184,15 @@ def convert_html_fields_in_state(cls, state_dict, conversion_fn):
                     'choices']['value'][value_index] = conversion_fn(value)
 
         return state_dict
+
+    def get_all_html_content_strings(self):
+        """Get all html content strings in the state.
+
+        Returns:
+            list(str). The list of all html content strings in the interaction.
+        """
+        html_list = (
+            self.written_translations.get_all_html_content_strings() +
+            self.interaction.get_all_html_content_strings() + [
+                self.content.html])
```

Comment:
Make sure that this all the html strings present in exploration.

Reply:
Yeah, I have verified them.